examples/example-xds/build.gradle

```diff
@@ -19,25 +23,46 @@ targetCompatibility = 1.7
 // Feel free to delete the comment at the next line. It is just for safely
 // updating the version in our release process.
 def grpcVersion = '1.34.0-SNAPSHOT' // CURRENT_GRPC_VERSION
+def nettyTcNativeVersion = '2.0.31.Final'
+def protocVersion = '3.12.0'
 
 dependencies {
-    // This example's client is the same as the helloworld client. We depend on the helloworld
-    // client's code here
-    implementation ':examples'
-    // The only change necessary is an extra runtime dependency on io.grpc:grpc-xds
-    runtimeOnly "io.grpc:grpc-xds:${grpcVersion}"
+    implementation "io.grpc:grpc-netty:${grpcVersion}"
```

Comment:
Users should generally be using grpc-netty-shaded, not grpc-netty.

Reply:
example-tls/build.gradle uses grpc-netty and not grpc-netty-shaded. That one also needs to be changed, I guess

Reply:
That example actually needed the netty APIs, so grpc-netty was appropriate.